packages/devextreme/js/__internal/grids/new/grid_core/toolbar/view.test.ts

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable spellcheck/spell-checker */
+/* eslint-disable @typescript-eslint/dot-notation */
+import { describe, expect, it } from '@jest/globals';
+
+import type { Options } from '../options';
+import { OptionsControllerMock } from '../options_controller/options_controller.mock';
+import { ToolbarController } from './controller';
+import { ToolbarView } from './view';
+
+const createToolbarView = (options?: Options): {
+  rootElement: HTMLElement;
+  optionsController: OptionsControllerMock;
+} => {
+  const rootElement = document.createElement('div');
+  const optionsController = new OptionsControllerMock(options ?? {
+    toolbar: {
+      visible: true,
+    },
+  });
+
+  const toolbarController = new ToolbarController(optionsController);
+  const toolbar = new ToolbarView(toolbarController, optionsController);
+
+  toolbar.render(rootElement);
+
+  return {
+    rootElement,
+    optionsController,
+  };
+};
+
+describe('render', () => {
+  it('empty toolbar', () => {
```

Comment:
Let's follow the pattern `describe('some entity in code')` -> `describe('some case')` -> `it('should ...')`
there's no entity `"render"` in code, so let's rewrite it as 
`describe("Toolbar")` -> `describe("when it is empty") -> `describe("it should be rendered correctly")`
And if it possible, instead of "should be rendered correctly" let's use something more specific, like "should be rendered to empty div" or "should not be rendered"